README.md

```diff
@@ -10,3 +10,17 @@ This will call pdflatex separately and compile everything into a final output `t
 If you want to rename, you should rename both the `.gp` and `.tex` files so they keep the same name.
 
 
+Add 
+
+
+```alias mkfig="sh /../figure-template/mkfig.sh
```

Comment:
I think we shouldn't rely on absolute path, nor on people using zsh. Simply say that there is a script called `mkfig.sh` that can be placed somewhere in the `PATH`, and describe its use.